### ISSUE TITLE
Feedback about first statement.

### ISSUE THREAD

**Issue by gfant** [allinbits/carmel#1]
## About Initial terms

> Both parties agree on a moderator.

There should be considered a replacement for the moderator in case of a problem resulting from the chosen moderator like a decease or involvement with any of the parties by any reason to let the affected party to remove him.

## Step 1. Determine the topic points.

> If there was a court filing, the majority of key claims must be from filings.

If a filling has more than 7 points how must be solved? Will the moderator have the responsibility to filter the relevant ones?

## Step 2. Debate each topic point.

> The defendant replies, or the court filing's replies are used (if present).

If it's not present, how the debate will follow after the first answer?

> Then plaintiff replies, then defendant replies, and so on until the moderator terminates the debate.

What happens if one of the parties doesn't respond?

> The medium of communication will be via email.

Why not a place where the messages will be encrypted so only the members can decrypt it to avoid any party to accuse the other party is the guilty of a slow debate? Each encrypted signature would mean the owner party of that signature has released another response.

> Each reply is expected to take some time to complete.

Maybe the moderator and the parties should agree on a time limit to respond before starting to avoid any party to take advantage of time in debates with short time to be developed.

## Extra A. Incentives, Dismissal, Releases

> Either party can incentivize for the other party to participate.

Talking about incentives the moderator should be considered. This is in order to keep a compromise with the debate until the finalisation.

> Incentives may include clawback clauses if the moderator determines appropriate.

This point reinforces the idea of a secondary moderator in case the first moderator becomes a bad actor for one of the parties.

## Extra B. Truthfulness

> If a false statement is made, punishable up to 2x the reward.

If a statement is proven to be false, shouldn't we consider mentioning it in the final statement to show the attempt of one of the parties if was bad intentioned. 

> At any time, either party can create a platform for, or run, a bounty program for whistleblowers to testify about false statements made.

What happens if the whistleblower has a relationship with the affected party? Is there a way to protect the whistleblowers? 

> The moderator decides the rules needed to preserve privacy where appropriate.

Does this point considers the privacy of the identity of the whistleblower?

> The moderator decides the rules needed to preserve privacy where appropriate.

Which are the consequences of not following the privacy set by the moderator?

**Comment by jaekwon** [comment]
> ## About Initial terms
> 
> > Both parties agree on a moderator.
> 
> There should be considered a replacement for the moderator in case of a problem resulting from the chosen moderator like a decease or involvement with any of the parties by any reason to let the affected party to remove him.

Yes. Also, what to do when two people can't agree on a moderator?
In the case of making a binary decision, there is a way to handle this:

### Safe & Live Binary Choice
 1. NumDelegates = 1
 2. Each side starts with 1 delegate each
 3. Each delegate must not be restricted by any relevant agreement or covenant
 4. If the delegates cannot reach a 51% agreement, NumDelegates *= 2 and goto 2.

This could work for choosing a singular moderator, but it's got enough wiggle room for social manipulation, so this isn't ideal.

One proposal is to let a third party select a moderator given some options, but the third party would have to be neutral and independent, so the cosmoshub doesn't work (nor is cosmoshub governance designed yet for choosing among choices; and this is a difficult problem, see "concordet criterion").

This would be better:

### Safe & Live Candidate Selection
 1. NumDelegates = 1
 2. Each side selects up to 3 moderator candidates ranked by order
 3. If there are any overlaps, they are moved to the approved_moderator_candidates set
 4. If there are no other candidates, and there is only 1 approved_moderator_candidate, done
 5. Of the remaining moderator candidates, trim the longer list so both have equal size
 6. Each side starts with 1 delegate each
 7. Each delegate must not be restricted by any relevant agreement or covenant
 8. If the delegates cannot select a singular winner via a voting mechanism that satisfies concordet criterion among the approved_moderator_candidates, NumDelegates *= 2 and goto 2.

But I wonder if it can be made simpler (or if there are any issues).

> ## Step 1. Determine the topic points.
> 
> > If there was a court filing, the majority of key claims must be from filings.
> 
> If a filling has more than 7 points how must be solved? Will the moderator have the responsibility to filter the relevant ones?

Each side must limit to 7 points, so must trim.
The moderator has no say until later, must not interact in the process of choosing.
(maybe the moderator should be chosen after this step).

> ## Step 2. Debate each topic point.
> 
> > The defendant replies, or the court filing's replies are used (if present).
> 
> If it's not present, how the debate will follow after the first answer?

If the defendant has no response filing, then the defendant makes up the first answer. 

> > Then plaintiff replies, then defendant replies, and so on until the moderator terminates the debate.
> 
> What happens if one of the parties doesn't respond?

If they don't respond, moderator may determine that the party is not cooperating.

> > The medium of communication will be via email.
> 
> Why not a place where the messages will be encrypted so only the members can decrypt it to avoid any party to accuse the other party is the guilty of a slow debate? Each encrypted signature would mean the owner party of that signature has released another response.

It doesn't have to be email (oops that's left over from design decision specifically for our dropped lawsuit, so this should ultimately be some open source crypto tool (on chain could work, but forward security could be an issue).

The problem with on-chain and your proposal is that it leaks/publishes metadata about who responded when, and it's probably better to not have the public speculate and cause drama.

> > Each reply is expected to take some time to complete.
> 
> Maybe the moderator and the parties should agree on a time limit to respond before starting to avoid any party to take advantage of time in debates with short time to be developed.

Yes something about the expected time would be good, and a soft limit where if exceeded, the moderator has option to do something about it.

> ## Extra A. Incentives, Dismissal, Releases
> 
> > Either party can incentivize for the other party to participate.
> 
> Talking about incentives the moderator should be considered. This is in order to keep a compromise with the debate until the finalization.

The incentive in this mode of Carmel is to get the other party to agree to participate to this in the first place, so it would have to happen before moderator selection. Additional incentives can be moderated.

> > Incentives may include claw-back clauses if the moderator determines appropriate.
> 
> This point reinforces the idea of a secondary moderator in case the first moderator becomes a bad actor for one of the parties.

I agree that we need a way to deal with bad moderators in general, but I don't see the need for a second moderator specifically for this step; but there might be issues with timing where the first moderator isn't available because claw back might happen way after conclusion of the debate.

> ## Extra B. Truthfulness
> 
> > If a false statement is made, punishable up to 2x the reward.
> 
> If a statement is proven to be false, shouldn't we consider mentioning it in the final statement to show the attempt of one of the parties if was bad intention-ed.

If the moderator agrees. It might not affect the final outcome, but we still need to punish to prevent selective lying in the first place, though the penalty might be smaller, and no amendment necessary.

> > At any time, either party can create a platform for, or run, a bounty program for whistleblowers to testify about false statements made.
> 
> What happens if the whistleblower has a relationship with the affected party? Is there a way to protect the whistleblowers?

Both parties must pre-agree to indemnify all whistleblowers who submit relevant information... as judged by the moderator? Some improvement to be made here.

> > The moderator decides the rules needed to preserve privacy where appropriate.
> 
> Does this point considers the privacy of the identity of the whistleblower?

Yes.

> Which are the consequences of not following the privacy set by the moderator?

Claw back as determined by the moderator.

**Comment by gfant** [comment]
I agree with you that the binary decision would not be the best method. Also it could take a lot of human effort for the mere choice of a moderator. In the case of the Candidate Selection method I would appreciate you explain it better so anyone reading this can understand it.

>  (maybe the moderator should be chosen after this step).

This makes sense because to have a debate you need to have a topic. And if you have a topic it means you have clear what you want to talk about.

> It doesn't have to be email (oops that's left over from design decision specifically for our dropped lawsuit, so this should ultimately be some open source crypto tool (on chain could work, but forward security could be an issue).

> The problem with on-chain and your proposal is that it leaks/publishes metadata about who responded when, and it's probably better to not have the public speculate and cause drama.

If you're looking for a crypto tool you will always drop some metadata. It's part of the txs. If the issue here is literally when a response was sent and the content, the only idea I have would be making both parties send junk messages (messages without the response) and when they want to send the response, they can include it in the next message. Since (I assume) both parties have the corresponding deciphering keys, only they and the moderator will notice there was an answer when they decrypt the message and receive a readable answer, and the public will know it after a pseudorandom number of junk posts. If someone asks when were the answers done, the realm can point the txs that relate to the conversation. 

But this sounds extremely overcomplicated in matter of txs for the chain and constant activity for the parties.
